IFC4x3/Sections/Core data schemas/Schemas/IfcProductExtension/Entities/IfcSpace/Documentation.md

```diff
@@ -1,14 +1,14 @@
-A space represents an area or volume bounded actually or theoretically. Spaces are areas or volumes that provide for certain functions within a building.
+A space represents an area or volume bounded actually or theoretically. Spaces are areas or volumes that provide for certain functions within a facility.
 
-A space is associated to a building storey (or in case of exterior spaces to a site). A space may span over several connected spaces. Therefore a space group provides for a collection of spaces included in a storey. A space can also be decomposed in parts, where each part defines a partial space. This is defined by the _CompositionType_ attribute of the supertype _IfcSpatialStructureElement_ which is interpreted as follow:
+A space is associated to a building storey, facility part or a facility (or in case of exterior spaces to a site). A space may span over several connected spaces. Therefore a space group provides for a collection of spaces included in a storey or facility part. A space can also be decomposed in parts, where each part defines a partial space. This is defined by the _CompositionType_ attribute of the supertype _IfcSpatialStructureElement_ which is interpreted as follow:
```

Comment:
Would it not make more sense to just use IfcExternalSpatialElement for this?

Reply:
thats another bag of worms, space PDTs are both exterior and interior and there is a property (in Pset_SpaceCommon that asks IsExternal) for defining this, so when is it a space or an IfcExternalSpatialElement. 

I dont see any reasonable use case where there is a need for an entity level split between IfcSpatialStructureElement (and subtypes) and IfcExternalSpatialStructureElement. prime example IfcSpatialElement only defines the delimitation between IfcSpatialStructureElement and IfcSpatialZone

Reply:
I understand your point. Perhaps we should then look into deprecating the branch of external elements?

Reply:
Yes, I think that is necessary, but is a larger conversation that will require a short group discussion, we have a small cross domain working group looking at some spatial subjects we can add this to that agenda possibily.

For now can we accept the documentation update on this pull request and sort out External Elements as a seperate task?
if you agree please resolve the convo.

Reply:
I would prefer the change to documentation as a separate pull request if it is not too much trouble. Let me know if this is possible.

Reply:
ok, i will seperate it out and attach it to my issue